core/node/libp2p/host.go

```diff
@@ -43,9 +46,21 @@ func Host(mctx helpers.MetricsCtx, lc fx.Lifecycle, params P2PHostIn) (out P2PHo
 	}
 
 	ctx := helpers.LifecycleCtx(mctx, lc)
+	cfg, err := params.Repo.Config()
+	if err != nil {
+		return out, err
+	}
+	var bootstrappers []ma.Multiaddr
+	for _, addrStr := range cfg.Bootstrap {
+		addr, err := ma.NewMultiaddr(addrStr)
+		if err != nil {
+			return out, fmt.Errorf("failed to parse bootstrap peer address %q: %s", addrStr, err)
+		}
+		bootstrappers = append(bootstrappers, addr)
+	}
```

Comment:
Changing `dht.BootstrapPeers` to take `peer.AddrInfo` would make this code simpler...

Reply:
@Stebalien Have made the change here:
https://github.com/libp2p/go-libp2p-kad-dht/pull/651

Also, we should now use `go-libp2p-kad-dht.GetDefaultBootstrapPeerAddrInfos` to set the default for `cfg. Bootstrap`.

Reply:
@Stebalien Have made the change here:
libp2p/go-libp2p-kad-dht#651

Also, we should now use go-libp2p-kad-dht.GetDefaultBootstrapPeerAddrInfos to set the default for cfg. Bootstrap.

Reply:
I'm not sure I really understand what makes this easier, but also no complaints from me :)

Reply:
It's easier in this case because `cfg` has a special function to get AddrInfos.